app/controllers/users_controller.rb

```diff
@@ -115,6 +115,15 @@ def profile
     @notes = DrupalNode.page(params[:page])
                        .order("nid DESC")
                        .where(type: 'note', status: 1, uid: @user.uid)
+                       .joins(:drupal_tag)
+                       .where('term_data.name NOT LIKE ?', 'question:%')
+                       .group('node.nid')
```

Comment:
@jywarren This query is not working as expected. I wan't to exclude any questions from the research notes section. But some of the questions are still included with this query. Some help regarding this would be great.